doc/source/whatsnew/v0.21.0.txt

```diff
@@ -90,6 +90,8 @@ Performance Improvements
 Bug Fixes
 ~~~~~~~~~
 
+- Fixes regression in 0.20, `aggregate` allows dictionaries as return values again (:issue:`16741`)
```

Comment:
be a little more verbose here. say :func:`Series.aggregate` and :func:`DataFrame.aggregate`